docs/css/remove.md

```diff
@@ -1,4 +1,4 @@
 ## Removing `sanitize.css`
 
-Delete [lines 31 and 32 in `app.js`](../../app/app.js#L31-L32) and remove it
+Delete [line 19 in `app.js`](../../app/app.js#L19) and remove it
 from the `dependencies` in [`package.json`](../../package.json)!
```

Comment:
Would be easier to maintain going forward if we gave example of code to be removed and let the dev find it themselves. As demonstrated here, line numbers change...